bower.json

```diff
@@ -34,12 +34,12 @@
     "angular-resource": "1.5.11",
     "angular-sanitize": "1.5.11",
     "angular-route": "1.5.11",
-    "angular-bootstrap": "0.14.3",
-    "angular-patternfly": "3.24.0",
+    "angular-bootstrap": "2.2.x",
```

Comment:
Fixed the angular-bootstrap issue in console, think we're good to go.  Thanks for the tip!